### PR TITLE
UTM Package

### PR DESCRIPTION
**Pull request by snowman2**
Initial commit. Waiting on maintainer question here: https://github.com/Turbo87/utm/issues/28.

### CODE REVIEW

File: recipes/utm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
```
Comment (synapticarbors) [inline recipes/utm/meta.yaml]: It appears as if you are missing the `build` section of `requirements`.  You probably minimally need python and setuptools.

File: recipes/utm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  license_file: LICENSE
+  summary: 'Bidirectional UTM-WGS84 converter for python.'
+  description: 'Bidirectional UTM-WGS84 converter for python.'
+  doc_url: https://github.com/Turbo87/utm
+  dev_url: https://github.com/Turbo87/utm
+
+extra:
+  recipe-maintainers:
+    - snowman2
+    - Turbo87
```
Comment (synapticarbors) [inline recipes/utm/meta.yaml]: @Turbo87 -- are you ok with being added as a co-maintainer of the utm conda package on conda-forge?

Comment (Turbo87) [inline recipes/utm/meta.yaml]: 👍

File: recipes/utm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/utm/meta.yaml]: It looks like the `setup.py` doesn't use `setuptools`, you just use `python setup.py install` and drop `setuptools` from the build requirements.

**Comment by snowman2** [comment]
@synapticarbors, thanks for your helpful comments!

**Pull request merged**